packages/ipfs-core/src/runtime/libp2p-nodejs.js

```diff
@@ -49,7 +49,7 @@ module.exports = () => {
       },
       dht: {
         kBucketSize: 20,
-        enabled: false,
+        enabled: true,
```

Comment:
This should be left as false.

Reply:
yes, sorry about that. I had other staged changes that I did not remember about